envelope.md

```diff
@@ -2,12 +2,18 @@
 
 May 10, 2024
 
-Version 1.0.1
+Version 1.0.2
 
 This document describes the recommended data structure for storing DSSE
 signatures, which we call the "JSON Envelope". For the protocol/algorithm, see
 [Protocol](protocol.md).
 
+**Important:** Implementations MUST ensure that the same payload bytes that are
```

Comment:
Does this belong further down? In / around the parsing rules? At this point in the doc, there isn't a mention of the `payload` field yet.

Reply:
Moved. PTAL